src/traces/sankey/attributes.js

```diff
@@ -185,6 +190,21 @@ module.exports = overrideAll({
             role: 'info',
             description: 'A numeric value representing the flow volume value.'
         },
+        hoverinfo: {
+            valType: 'enumerated',
+            values: ['all', 'none', 'skip'],
+            dflt: 'all',
+            role: 'info',
+            description: [
+                'Determines which trace information appear when hovering links.',
+                'If `none` or `skip` are set, no information is displayed upon hovering.',
+                'But, if `none` is set, click and hover events are still fired.'
+            ].join(' ')
+        },
+        hoverlabel: fxAttrs.hoverlabel, // needs editType override,
         description: 'The links of the Sankey plot.'
     }
 }, 'calc', 'nested');
+// hide unsupported top-level properties from plot-schema
+attrs.hoverinfo = undefined;
+attrs.hoverlabel = undefined;
```

Comment:
But wait, trace `hoverlabel` was working before this. Hmm, I'd call this a breaking change.

Reply:
... I think we should either make node/link `hoverlabel` inherit from trace `hoverlabel` or add some [`cleanData`](https://github.com/plotly/plotly.js/blob/7ae6fd68533d6489e3161cdbdfe2a44217f530e1/src/plot_api/helpers.js#L206) logic.

Reply:
@etpinard It's great that you noticed that change and I agree this should be handled better! 

FYI, having a `hover(label|info)` in each `node` and `link` was the fruit of discussions with @alexcjohnson. In the future, we could add a `hovertemplate` on each to provide more control. That's one way of dealing with point 2 of this comment https://github.com/plotly/plotly.js/pull/3126#issue-224077784 which we can discuss about later on :)

Reply:
Ah indeed - good catch @etpinard - I'd go for:

> make node/link `hoverlabel` inherit from trace `hoverlabel`

That would be more friendly for the (probably most common) case that users want node and link hover labels styled the same.

Reply:
... and maybe we should have a trace `hoverinfo` that acts similarly.

Reply:
`hoverinfo` is trickier - could be nice to inherit `skip`/`none`, but when we get around to supporting a real flaglist, links and nodes will have completely different options.

Reply:
e.g. `trace.hoverinfo: 'none'` would be equivalent to `node.hoverinfo: 'none'` + `link.hoverinfo: 'none'`.

Not all the flags to be implemented in https://github.com/plotly/plotly.js/issues/3097 should be available from the trace hoverinfo, but I think setting trace-wide `'none'` and `'skip'` could be useful.